integration/src/integration/java/pl/allegro/tech/hermes/integration/AbstractFrontendShutdownTest.java

```diff
@@ -8,6 +8,10 @@
 import pl.allegro.tech.hermes.test.helper.endpoint.HermesPublisher;
 import pl.allegro.tech.hermes.test.helper.util.Ports;
 
+import static pl.allegro.tech.hermes.common.config.Configs.KAFKA_AUTHORIZATION_ENABLED;
```

Comment:
I think that this static should be from `pl.allegro.tech.hermes.integration.ConfigurationProperties`